packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -995,8 +999,8 @@ export class AttributesController extends ConsumptionBaseController {
             return ValidationResult.error(ConsumptionCoreErrors.attributes.successionMustNotChangeValueType());
         }
 
-        if (predecessor.hasDeletionInfo() && predecessor.deletionInfo.deletionStatus !== LocalAttributeDeletionStatus.DeletionRequestRejected) {
-            return ValidationResult.error(ConsumptionCoreErrors.attributes.cannotSucceedAttributesWithDeletionInfo());
+        if (successor.succeeds && !predecessorId.equals(successor.succeeds.toString())) {
+            return ValidationResult.error(ConsumptionCoreErrors.attributes.setPredecessorIdDoesNotMatchActualPredecessorId());
```

Comment:
Why did you put this at the bottom? I'm not sure if it's possible but could there be a way that the content check fails and then the user tries to tweak with the content only to run into this error?

Reply:
@Magnus-Kuhn That could in fact happen. But if there is an issue with the content and the manually set predecessorId, the second error will always be thrown fixing the first, no matter which way I put them. And since I think that fixing the content is more important, while simply omitting the manually set predecessorId is super simple, I decided to place them this way around.